docs/api.rst

```diff
@@ -103,6 +103,8 @@ Data Utils
    :toctree: generated/
 
     create_from_X_y
+    create_trials_from_X_y
+    create_windows_from_X_y
```

Comment:
would it make sense to keep a single `create_from_X_y` and then internally, dependings with items of X have the same length use windows or base? Basically as a user do I need to care of I can just pass the chunks of EEG data I have an the models will just deal with them? if you expect some breakage as some models only work with WindowsDataset it's another story but if you expect models to 'just work' maybe we can avoid some complexity on the user side.

Reply:
Ok, I will try to explain what's going on. We just had a usecase (see https://github.com/braindecode/braindecode/issues/136) where a user had a dataset in Xy format, however, it was not preprocessed. So he wanted to convert it into braindecode format to be able to use the available braindecode/mne preprocessing. So far, the ```create_from_X_y``` function assumed data was already preprocessed and directly transformed it into compute windows for decoding. So it did not allow for preprocessing. I suggested something similar to you (see https://github.com/braindecode/braindecode/issues/136#issuecomment-662299096), @robintibor did not agree. It is still undecided / open for discussion.